src/instructlab/config/show.py

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: Apache-2.0
+import click
+import yaml
+
+from instructlab import utils
+
+
+@click.command()
+@click.pass_context
+@utils.display_params
+def show(
+    ctx,
+):
+    """Displays the current config as YAML"""
+    # TODO: make this use pretty colors like jq/yq
+    config_yaml = yaml.load(ctx.obj.config.model_dump_json(), Loader=yaml.FullLoader)
```

Comment:
What if `yaml.load` fail? How is the error handled?

Reply:
Added an exception handle